pkg/connector/backfill.go

```diff
@@ -98,6 +98,7 @@ func (l *LinkedInClient) FetchMessages(ctx context.Context, fetchParams bridgev2
 			Sender:           sender,
 			ID:               msg.MessageID(),
 			Timestamp:        msg.DeliveredAt.Time,
+			StreamOrder:      msg.DeliveredAt.UnixMilli(),
```

Comment:
Did you check that the native timestamps are milliseconds? (we should use the native precision for stream orders)